mollie-payments-for-woocommerce/includes/mollie/wc/plugin.php

```diff
@@ -216,12 +217,38 @@ public static function init() {
 		// Capture order at Mollie (for Orders API/Klarna)
 		add_action( 'woocommerce_order_status_completed', array( __CLASS__, 'shipAndCaptureOrderAtMollie' ) );
 
+        // Enqueue Scripts
+        add_action('wp_enqueue_scripts', [__CLASS__, 'enqueueFrontendScripts']);
+
 		self::initDb();
 		self::schedulePendingPaymentOrdersExpirationCheck();
 		// Mark plugin initiated
 		self::$initiated = true;
 	}
 
+    /**
+     * Enqueue Frontend only scripts
+     *
+     * @return void
+     */
+    public static function enqueueFrontendScripts()
+    {
+        if (is_admin()) {
+            return;
+        }
+
+        if (is_checkout()) {
+            wp_register_script(
```

Comment:
It would be great if all scripts and styles could be _registered_ elsewhere unconditionally, and then conditionally _enqueued_ e.g. here.

Reply:
@XedinUnknown There's no way defined where the scripts are registered or enqueued. 
That could be discussed, may be adding a library to manage them but for now is it as it is.